pkg/cloud/services/network/secondarycidr.go

```diff
@@ -17,12 +17,21 @@ limitations under the License.
 package network
 
 import (
+	"reflect"
+
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/pkg/errors"
 
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/record"
 )
 
+func isVPCPresent(vpcs *ec2.DescribeVpcsOutput) error {
```

Comment:
A function prefixed with `is` implies to be that it tests something and returns a boolean. 

So could it be something like this:

```go
func isVPCPresent(vpcs *ec2.DescribeVpcsOutput) bool {
  return vpcs != nil && len(vpcs.Vpcs) > 0
}
```

Reply:
@shivi28 - what do you tthink to this ^^^